Jenkinsfile-candidate

```diff
@@ -0,0 +1,34 @@
+#!groovy
+node {
+
+    load "$JENKINS_HOME/jobvars.env"
+
+    env.JAVA_HOME = "${tool 'openjdk-11'}"
+    env.PATH = "${env.JAVA_HOME}/bin:${env.PATH}"
+
+    stage('Checkout') {
+        checkout scm
+    }
+    stage('Assemble') {
+        sh "./gradlew clean assemble -P buildNumber=${env.BUILD_NUMBER}"
+    }
+    stage('Test') {
+        sh './gradlew test --full-stacktrace'
+    }
+    stage('Build') {
+        sh './gradlew build'
```

Comment:
-x test? 
It looks like tests will be executed twice

Reply:
fixed